v_frame/src/frame.rs

```diff
@@ -27,8 +27,9 @@ impl<T: Pixel> Frame<T> {
     width: usize, height: usize, chroma_sampling: ChromaSampling,
     luma_padding: usize,
   ) -> Self {
-    let luma_width = width.align_power_of_two(3);
-    let luma_height = height.align_power_of_two(3);
+    //TODO: If SB size is 128x128, 7 should be used instead of 6
```

Comment:
Hi,

I was thinking whether we should keep separate commit for v_frame changes as it is a separate module and we do maintain releases for them.
If it is split as an extra commit for v_frame changes, we can make things easier,

But I am not sure if it will break the intermediate commits or not, if that is the case, we should be having this style only,

Reply:
This change has been removed.